Shoko.Server/API/Startup.cs

```diff
@@ -70,6 +70,7 @@ public void ConfigureServices(IServiceCollection services)
                 j => j.WithIdentity(ConnectivityMonitorJob.Key).DisallowConcurrentExecution().Build());
 
             q.AddJob<ImportJob>(j => j.WithIdentity(ImportJob.Key).DisallowConcurrentExecution().StoreDurably().Build()); // TODO: Maybe add schedule
+            q.AddJob<ScanFolderJob>(j => j.WithIdentity(ScanFolderJob.Key).DisallowConcurrentExecution().StoreDurably().Build()); // TODO: Maybe add schedule
```

Comment:
@da3dsoul thought - should we make our own extension to have a lot of these added without the same config all the time?